src/test/java/it/gov/pagopa/arc/dto/mapper/bizevents/paidnotice/BizEventsPaidNoticeDTO2NoticeDTOTest.java

```diff
@@ -0,0 +1,86 @@
+        //when
+        NoticeDTO dtoMapped = mapper.toNoticeDTO(bizEventsPaidNoticeDTO);
+        //then
+        commonAssert(bizEventsPaidNoticeDTO, dtoMapped);
+        assertEquals(MapperUtilities.euroToCents(bizEventsPaidNoticeDTO.getAmount()), dtoMapped.getAmount());
+    }
+    @Test
+    void givenBizEventsPaidNoticeWhenIsCartIsTrueThenReturnNoticeDTO(){
+        //given
+        BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO = BizEventsPaidNoticeDTOFaker.mockInstance(1, true);
+        //when
+        NoticeDTO dtoMapped = mapper.toNoticeDTO(bizEventsPaidNoticeDTO);
+
+        //then
+        commonAssert(bizEventsPaidNoticeDTO, dtoMapped);
+        assertNull(dtoMapped.getAmount());
+    }
+
+    @Test
+    void givenBizEventsPaidNoticeWhenAmountAndDateAreNullThenReturnNoticeDTO(){
+        //given
+        BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO = BizEventsPaidNoticeDTOFaker.mockInstance(1, false);
+        bizEventsPaidNoticeDTO.setAmount(null);
+        bizEventsPaidNoticeDTO.setNoticeDate(null);
+        //when
+        NoticeDTO dtoMapped = mapper.toNoticeDTO(bizEventsPaidNoticeDTO);
+
+        //then
+        assertEquals(bizEventsPaidNoticeDTO.getEventId(), dtoMapped.getEventId());
+        assertEquals(bizEventsPaidNoticeDTO.getPayeeName(), dtoMapped.getPayeeName());
+        assertEquals(bizEventsPaidNoticeDTO.getPayeeTaxCode(), dtoMapped.getPayeeTaxCode());
+        assertEquals(bizEventsPaidNoticeDTO.getIsCart(), dtoMapped.getIsCart());
+        assertEquals(bizEventsPaidNoticeDTO.getIsPayer(), dtoMapped.getPaidByMe());
+        assertEquals(bizEventsPaidNoticeDTO.getIsDebtor(), dtoMapped.getRegisteredToMe());
+        assertNull(dtoMapped.getAmount());
+        assertNull(dtoMapped.getNoticeDate());
+    }
+
+    @Test
+    void givenBizEventsPaidNoticeListDTOWhenCallMapperThenReturnListOfNoticeDTO() {
+        //given
+        BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO = BizEventsPaidNoticeDTOFaker.mockInstance(1, false);
+        BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO2 = BizEventsPaidNoticeDTOFaker.mockInstance(2, false);
+        BizEventsPaidNoticeListDTO bizEventsPaidNoticeListDTO = BizEventsPaidNoticeListDTO.builder().notices(List.of(bizEventsPaidNoticeDTO, bizEventsPaidNoticeDTO2)).build();
+        //when
+        List<NoticeDTO> noticeDTOList = mapper.toNoticeDTOList(bizEventsPaidNoticeListDTO);
+        //then
+        commonAssert(bizEventsPaidNoticeDTO, noticeDTOList.get(0));
+        commonAssert(bizEventsPaidNoticeDTO2, noticeDTOList.get(1));
+        assertEquals(MapperUtilities.euroToCents(bizEventsPaidNoticeDTO.getAmount()), noticeDTOList.get(0).getAmount());
+        assertEquals(MapperUtilities.euroToCents(bizEventsPaidNoticeDTO2.getAmount()), noticeDTOList.get(1).getAmount());
+
+    }
+
+    private static void commonAssert(BizEventsPaidNoticeDTO bizEventsPaidNoticeDTO, NoticeDTO dtoMapped) {
```

Comment:
I would add a check to assert if all the fields are set on the result.
This way you will verify that eventually new fields will be set...